<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mugi-query</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="sessionTree">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="sessionTabs">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="addDatabase"/>
    <addaction name="reconnect"/>
    <addaction name="removeDatabase"/>
    <addaction name="databaseHistory"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>&amp;Query</string>
    </property>
    <addaction name="queryHistory"/>
    <addaction name="saveData"/>
    <addaction name="queryHelp"/>
   </widget>
   <widget class="QMenu" name="menu_Session">
    <property name="title">
     <string>&amp;Session</string>
    </property>
    <addaction name="addSession"/>
    <addaction name="removeSession"/>
   </widget>
   <widget class="QMenu" name="menuData_2">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="dataSave"/>
    <addaction name="dataCopy"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuData"/>
   <addaction name="menu_Session"/>
   <addaction name="menuData_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="addDatabase">
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveData">
   <property name="text">
    <string>&amp;Save data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="databaseHistory">
   <property name="text">
    <string>&amp;HIstory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="queryHistory">
   <property name="text">
    <string>&amp;HIstory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="addSession">
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="removeSession">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="removeDatabase">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
  </action>
  <action name="reconnect">
   <property name="text">
    <string>Re&amp;connect</string>
   </property>
  </action>
  <action name="dataSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="dataCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="queryHelp">
   <property name="text">
    <string>H&amp;elp</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
